README.md

```diff
@@ -1,2 +1,154 @@
-# quasar-app-extension-meta
-Automatically manage social meta tags for websites projects hooking into Quasar Meta plugin
+# @dreamonkey/quasar-app-extension-meta
+
+This extension is made exclusively for [Quasar framework](https://quasar.dev/) and adds and simplify the use of their awesome [meta-plugin](https://quasar.dev/quasar-cli/developing-ssr/seo-for-ssr#Quasar-Meta-Plugin).
+
+# Why should i use this extension
+
+There are several reasons why you should use this [AppExtension](https://quasar.dev/app-extensions/introduction#Introduction) and let me introduce you why:
+
+1. Better SEO:
+   You know that SEO is not a simple task to accomplish and you also know that your best performance depends on that. How someone can find your page under other tonns? Well this mixin doesn't instantli put you page on top of that pile of results but it really helps you to scales up your visibility!
+
+2. It's an app extension:
```

Comment:
"Puoi installarla perchè funziona solo su Quasar" non è esattamente un pro, è solo un dato di fatto...
In generale, il fatto che tu possa installare le AE solo su Quasar è un dato che devi dare per assodato, dopo averlo esplicitato una volta all'inizio